### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by centipeda**
I wrote a draft of guidelines for prospective contributors here. Please comment if there are any changes you think should be made.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Contributing to Brick-a-Pic
+
+Welcome to the Brick-a-Pic project! Thank you for considering contributing. If you have any questions, please join our [Gitter]() or [open an issue]().
```
Comment (stephen-hannon) [inline CONTRIBUTING.md]: Looks like these are missing links? Also ESlint under "Code style"

Comment (centipeda) [reply CONTRIBUTING.md]: Whoops, I guess those didn't make it in. Will do.

Comment (centipeda) [reply CONTRIBUTING.md]: Added links.

**Review by stephen-hannon — APPROVED** [review]
Looks good, thanks for doing this

**Pull request merged**